corehq/messaging/smsbackends/airtel_tcl/models.py

```diff
@@ -105,7 +105,11 @@ def get_phone_number(phone_number):
         """
         phone_number = strip_plus(phone_number)
         if phone_number.startswith('91') and len(phone_number) > 2:
-            return phone_number[2:]
+            pattern = '^[6-9]'
+            result = re.match(pattern, phone_number[2:])
+            if result:
+                return phone_number[2:]
+
```

Comment:
Can you update tests https://github.com/dimagi/commcare-hq/blob/579b52f1037d30c8e9184214f7278bb1d6a8b9d2/corehq/messaging/smsbackends/airtel_tcl/tests/test_airtel_tcl.py#L112 ?